<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistrictEmployeeDashboard</class>
 <widget class="QMainWindow" name="DistrictEmployeeDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>707</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sunnydale School District - Admin Portal</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* === GENERAL APP STYLE === */
QMainWindow {
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
    color: #333;
}

QWidget {
    background-color: rgb(255, 252, 228);
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
    color: #333;
}

/* === PUSH BUTTONS === */
QPushButton {
    background-color: rgb(5, 60, 171);
    color: white;
    border: none;
    border-radius: 6px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #357ABD;
}

QPushButton:pressed {
    background-color: #2e5aa0;
}

/* === COMBOBOX, INPUTS, TEXTAREA === */
QComboBox, QLineEdit, QTextEdit, QDateEdit, QTimeEdit {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 4px 6px;
}

QComboBox::drop-down {
    border: none;
    background-color: white;
}

QComboBox QAbstractItemView {
    background-color: white;
}

/* === TABLES === */
QTableWidget, QTableView {
    background-color: #ffffff;
    alternate-background-color: #f0f0f0;
    gridline-color: #dcdcdc;
    border: 1px solid #ccc;
    border-radius: 4px;
}

QHeaderView::section {
    background-color: #e6e9ed;
    padding: 4px;
    border: none;
    font-weight: bold;
}

/* === LIST WIDGETS === */
QListWidget {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 4px;
}

/* === CALENDAR === */
QCalendarWidget QToolButton {
    color: #4a90e2;
}

QCalendarWidget QAbstractItemView {
    selection-background-color: #4a90e2;
    selection-color: white;
}

/* === GROUP BOXES === */
QGroupBox {
    border: 1px solid #aaa;
    border-radius: 6px;
    margin-top: 6px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 4px;
    background-color: transparent;
    color: #4a90e2;
    font-weight: bold;
}

/* === TABS === */
QTabWidget::pane {
    border: 1px solid #ccc;
    border-radius: 4px;
}

QTabBar::tab {
    background: #e6e9ed;
    padding: 6px 12px;
    border: 1px solid #ccc;
    border-bottom: none;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QTabBar::tab:selected {
    background: #ffffff;
    color: #4a90e2;
    font-weight: bold;
}

/* === TOOLTIP === */
QToolTip {
    background-color: #333;
    color: #fff;
    padding: 4px;
    border-radius: 4px;
}</string>
   </property>
   <widget class="QWidget" name="centralLayoutContainer">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>533</width>
      <height>550</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout"/>
   </widget>
   <widget class="QTabWidget" name="mainTabs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>951</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dashboardTab">
     <attribute name="title">
      <string>   Dashboard   </string>
     </attribute>
     <layout class="QVBoxLayout" name="dashboardLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QHBoxLayout" name="sunnydale_header">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="sunnydale_header_png">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>:/sunnydale_header.png/sunnydale_header.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logout_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI,sans-serif</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">
   	font-size: 15pt;
</string>
             </property>
             <property name="text">
              <string>Logout</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16pt;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Welcome, Admin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="kpiHeaderLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupTotalSchools">
          <property name="styleSheet">
           <string notr="true"> background-color: white;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="layoutTotalSchools">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI,sans-serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt;</string>
             </property>
             <property name="text">
              <string>Total Schools</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblTotalSchools">
             <property name="styleSheet">
              <string notr="true">font-size: 40px; font-weight: bold;</string>
             </property>
             <property name="text">
              <string>10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupActiveStudents">
          <property name="styleSheet">
           <string notr="true"> background-color: white;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="layoutActiveStudents">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI,sans-serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt;</string>
             </property>
             <property name="text">
              <string>Active Students</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblActiveStudents">
             <property name="styleSheet">
              <string notr="true">font-size: 40px; font-weight: bold; </string>
             </property>
             <property name="text">
              <string>3,000</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupAvgAttendance">
          <property name="styleSheet">
           <string notr="true"> background-color: white;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="layoutAvgAttendance">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI,sans-serif</family>
               <pointsize>18</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 18pt;</string>
             </property>
             <property name="text">
              <string>Absence Rate</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblAvgAttendance">
             <property name="styleSheet">
              <string notr="true">font-size: 40px; font-weight: bold;</string>
             </property>
             <property name="text">
              <string>3.57%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="filterLayout">
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="lblSchool">
          <property name="styleSheet">
           <string notr="true">font: 15pt;</string>
          </property>
          <property name="text">
           <string>School: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboSchool">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblGrade">
          <property name="styleSheet">
           <string notr="true">font: 15pt;</string>
          </property>
          <property name="text">
           <string>Grade Level: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboGrade">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="analysisLayout">
        <item>
         <layout class="QVBoxLayout" name="examLayout">
          <item>
           <widget class="QGroupBox" name="groupExam">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="examGroupLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>Segoe UI,sans-serif</family>
                 <pointsize>16</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>true</underline>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 16pt;</string>
               </property>
               <property name="text">
                <string>Exam Score Analysis</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout">
               <item>
                <widget class="QLabel" name="lblExamType">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 15pt;</string>
                 </property>
                 <property name="text">
                  <string>Exam Type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboExamType">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QComboBox::drop-down {
    border: none;
    background-color: white;
}</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Mid Exam</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Final Exam</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btnFetchExamStats">
               <property name="text">
                <string>Get Average Score</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblExamResult">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 18pt;</string>
               </property>
               <property name="text">
                <string>Avg: --</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="attendanceLayout">
          <item>
           <widget class="QGroupBox" name="groupAttendance">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="attendanceGroupLayout">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>Segoe UI,sans-serif</family>
                 <pointsize>16</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>true</underline>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 16pt;</string>
               </property>
               <property name="text">
                <string>Attendance Analysis</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout">
               <item>
                <widget class="QLabel" name="lblDate">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 15pt;</string>
                 </property>
                 <property name="text">
                  <string>Date:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="dateAttendance">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QComboBox::drop-down {
    border: none;
    background-color: white;
}</string>
                 </property>
                 <property name="dateTime">
                  <datetime>
                   <hour>0</hour>
                   <minute>0</minute>
                   <second>0</second>
                   <year>2025</year>
                   <month>3</month>
                   <day>18</day>
                  </datetime>
                 </property>
                 <property name="maximumDateTime">
                  <datetime>
                   <hour>23</hour>
                   <minute>59</minute>
                   <second>59</second>
                   <year>2025</year>
                   <month>5</month>
                   <day>15</day>
                  </datetime>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btnFetchAttendance">
               <property name="text">
                <string>Get Attendance Rate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblAttendanceResult">
               <property name="styleSheet">
                <string notr="true">font: 18pt;</string>
               </property>
               <property name="text">
                <string>Avg: --</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="executiveTab">
     <attribute name="title">
      <string>   Executive Dashboard   </string>
     </attribute>
     <layout class="QVBoxLayout" name="executiveTabLayout">
      <item>
       <widget class="QLabel" name="labelSchoolChartTitle">
        <property name="text">
         <string notr="true">&lt;span style=&quot;font-size:18pt; font-weight:600;&quot;&gt;Attendance Rate by School&lt;/span&gt;&lt;br&gt;&lt;span style=&quot;font-size:10pt; color:#666;&quot;&gt;(May 13, 2025)&lt;/span&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="schoolBarChart" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelSelectSchool">
          <property name="text">
           <string>Select School:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboSchoolChart"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelGradeChartTitle">
        <property name="text">
         <string notr="true">&lt;span style=&quot;font-size:18pt; font-weight:600;&quot;&gt;Grade-Level Attendance&lt;/span&gt;&lt;br&gt;&lt;span style=&quot;font-size:10pt; color:#666;&quot;&gt;(May 13, 2025)&lt;/span&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="gradeBarChart" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
